packages/core/src/Transform.ts

```
@@ -588,6 +577,24 @@ export class Transform extends Component {
     return isInvert;
   }
 
+  /**
+   * @internal
+   */
+  _generateTransitionState(state: TransformTransitionState): void {
+    state.position ? state.position.copyFrom(this._position) : (state.position = this._position.clone());
+    state.rotation ? state.rotation.copyFrom(this._rotation) : (state.rotation = this._rotation.clone());
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)